### DISCUSSION TITLE
Include workout instructions

### DISCUSSION THREAD

**Discussion by gloscherrybomb** [lcarcamo1526/TrainerRoad-to-Zwift#30] [General]
Hi, do you think it would be impossible to include the workout instructions so that they are shown on screen in zwift? I'd have a crack myself but I don't have the skills!

Btw this is awesome. I'd have cancelled my TR sub without it. Thanks.

**Comment by lcarcamo1526** [comment]
Hey @gloscherrybomb,

Sadly, I haven't used my trainer in a while and therefore do not use TR. However, can you give me the ID or the name of a workout that has instructions on it? I could take a look.

**Reply by gloscherrybomb** [reply]
Hi, thanks for looking in to it. Lots of TR workouts have instructions now that appear on the screen to guide the user through the workout. Some examples:
Homers nose
Taylor
Eisen
